src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponent.scala

```diff
@@ -111,7 +113,7 @@ trait EntityComponent {
 
     }
 
-    def entityAttributes(entityId: Long) = for {
+    def entityAttributes(entityId: Rep[Long]): EntityAttributeQuery = for {
```

Comment:
I needed to do this so I could call it from my for-comp. This function was unused until then anyway, and slick provides implicits for `Foo` to `Rep[Foo]` in most cases anyway.